hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/models/UpdateTrackingAccount.java

```diff
@@ -38,28 +38,29 @@
 import org.hyperledger.besu.evm.account.MutableAccount;
 
 /**
- * A mutable and updatable implementation of the {@link MutableAccount} interface, that tracks account updates
- * since the creation of the updated it's linked to.
- *
- * Contains {@code updateAccountTracker} for immediate set of balance in the world state.
- * Note that in practice this only track the modified account values, but doesn't remind if they were modified or not.
+ * A mutable and updatable implementation of the {@link MutableAccount} interface, that tracks account updates since the
+ * creation of the updated it's linked to.
+ * <p>
+ * Contains {@code updateAccountTracker} for immediate set of balance in the world state. Note that in practice this
+ * only track the modified account values, but doesn't remind if they were modified or not.
  */
 public class UpdateTrackingAccount<A extends Account> implements MutableAccount, EvmAccount {
+    @Nullable
+    protected final A account;
+
     private final Address address;
     private final Hash addressHash;
-    private long nonce;
-    private Wei balance;
-    private HederaEvmEntityAccess hederaEvmEntityAccess;
-    private boolean storageWasCleared = false;
     private final UpdateAccountTracker updateAccountTracker;
     private final NavigableMap<UInt256, UInt256> updatedStorage;
 
-    @Nullable
-    protected final A account;
-
     @Nullable
     protected Bytes updatedCode;
 
+    private long nonce;
+    private Wei balance;
+    private HederaEvmEntityAccess hederaEvmEntityAccess;
+    private boolean storageWasCleared = false;
```

Comment:
(Putting this comment here because Github won't let me put it on L67 and L80 where it belongs.)

I'd really prefer if the `updateAccountTracker` constructor args were **not** `@Nullable`.  It would make everything easier to read and understand if you could just always assume there was one to use.  Only _one_ caller in the main code (not tests - didn't look there) passes a null, that's at `AbstractLedgerEvmWorldUpdater.getAccount`.  I'd suggest a do-nothing implementation of the interface - in fact, it could even be _default_ do-nothing methods on the _interface_ I think, or at any rate a single static instance you get from the interface - that could be used for this. Cost is practically nothing, would be immeasurable in practice.  Things would be clearer to read.  I'd make the caller pass it in (as I said, there's only one such place now) but it _could_ be that the constructors set the `final` field here appropriately if null was passed in.

Reply:
I am not sure what is the purpose of @Nullable